core/src/perspectives/PerspectiveProxy.ts

```
@@ -1077,13 +1243,20 @@ export class PerspectiveProxy {
-            throw "No destructor found for given subject class: " + className
+
+        // Try SHACL links first
+        let actions = await this.getActionsFromSHACL(className, "ad4m://destructor");
+
+        if (!actions) {
+            // Fall back to Prolog
+            let result = await this.infer(`subject_class("${className}", C), destructor(C, Actions)`)
+            if(!result.length) {
+                throw "No destructor found for given subject class: " + className
+            }
+            actions = result.map(x => eval(x.Actions))[0]
```

Biome (2.3.13)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)